app/src/main/res/layout/fragment_relation.xml

```diff
@@ -24,4 +25,20 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/tlMentorshipRelation" />
 
+    <android.support.v7.widget.AppCompatTextView
+        android:id="@+id/tv_empty_state"
+        android:layout_width="250dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="8dp"
+        android:text="@string/no_current_mentorship_relation"
+        android:textAlignment="center"
+        android:textColor="@android:color/black"
```

Comment:
I'm not sure about the names of the classes but I think you got the idea. You sholdn't have to do 2 network calls for the same exact service, you could just pass on the object in case of sucess, to the tab responsible for the "Details". Do you understand? @mohak1283